src/Native/System.Security.Cryptography.Native/pal_x509_name.cpp

```diff
@@ -22,3 +22,76 @@ extern "C" X509_NAME* DecodeX509Name(const unsigned char* buf, int32_t len)
 
     return d2i_X509_NAME(nullptr, &buf, len);
 }
+
+extern "C" void X509NameDestroy(X509_NAME* a)
+{
+    if (a != nullptr)
+    {
+        X509_NAME_free(a);
+    }
+}
+
+static int32_t X509NamePrint(BIO* out, X509_NAME* nm, unsigned long flags)
```

Comment:
In my X500DN change I've removed the need for `X509NamePrint`, `X509NamePrintForFind`, `ConvertFormatFlags`, and `X509NamePrintEx`; just FYI.  (One of us is going to have some merging to do :smile:)